# extremely ugly workaround to compensate for missing CXXFLAGS in a test

--- configure.orig
+++ configure
@@ -16399,10 +16399,10 @@ else
 
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
 $as_echo "no" >&6; }
-        { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "Positional printf argument support in wxWidgets is required by FileZilla. Please rebuild wxWidgets and pass --enable-printfposparam to its configure script.
-See \`config.log' for more details" "$LINENO" 5; }
+#         { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
+# $as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+# as_fn_error $? "Positional printf argument support in wxWidgets is required by FileZilla. Please rebuild wxWidgets and pass --enable-printfposparam to its configure script.
+# See \`config.log' for more details" "$LINENO" 5; }
 
 
 fi
